bin/install-wp-tests.sh

```diff
@@ -12,8 +12,10 @@ DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
```

Comment:
I updated this file with the latest version from https://github.com/wp-cli/sample-plugin/tree/master/bin